### PR TITLE
[Mellanox] update for test_acl_tcp_rst due to design change

### PR DESCRIPTION
**Pull request by JibinBao**
### Description of PR

1. For receiver and sender, tcp rst packet will have no payload.
2. Remove some fields ignore to check them, because they are expected same as the sent packet.

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
update test_acl_tcp_rst 

#### How did you do it?
1. update rst packet lenght due to there is no payload.
2. Remove some fields ignore to check them, because they are expected same as the sent packet.

#### How did you verify/test it?
Run test_acl_tcp_rst  on mellanox dpu 

#### Any platform specific information?
Mellanox dpu

### CODE REVIEW

**Comment by bingwang-ms** [comment]
@Pterosaur Can you please review?

File: tests/dash/dash_acl.py [hunk 1]
```diff
@@ -1146,11 +1141,12 @@ def _get_expected_rst_packet_to_receiver():
             inner_extra_conf_to_receiver = copy.deepcopy(pkt.inner_extra_conf)
             inner_extra_conf_to_receiver["tcp_flags"] = "R"
             inner_extra_conf_to_receiver["ip_id"] = 0x0000
+            inner_extra_conf_to_receiver["pktlen"] = 54
```
Comment (Pterosaur) [inline tests/dash/dash_acl.py:1144]: Why is this number 54?

**Comment by JibinBao** [comment]
Hi @Pterosaur ,
If you have other comments, can you help approve it ? Thanks

**Comment by bingwang-ms** [comment]
@Pterosaur Can you approve if there is no other comments?

**Review by Pterosaur — APPROVED** [review]

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**